File: src/main/webapp/src/js/data/PointResult.js
class PointResult {
    constructor({x, y, r, result, timestamp, executionTime}) {
        this.x = x;
        this.y = y;
        this.r = r;
        this.result = result;
        this.timestamp = timestamp;
        this.executionTime = executionTime;
    }
}

export default PointResult;

File: src/main/java/ru/bardinpetr/itmo/lab3/data/models/PointResult.java
package ru.bardinpetr.itmo.lab3.data.models;

import jakarta.persistence.*;
import lombok.Data;

import java.io.Serializable;
import java.time.Duration;
import java.time.LocalDateTime;

@Data
@Entity
@Table(name = "point_result")
public class PointResult implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Embedded
    private Point point;

    @Embedded
    private AreaConfig area;

    @Column(nullable = false)
    private Boolean isInside;

    @Column(nullable = false)
    private LocalDateTime timestamp;

    @Column(nullable = false)
    private Duration executionTime;
}
